src/jdk.hotspot.agent/doc/clhsdb.html

```diff
@@ -51,7 +51,7 @@ <h3>Annotated output of CLHSDB help command</h3>
   dumpilt -a | id <font color="red">dump inline tree for C2 compilation</font>
   dumpreplaydata &lt;address&gt; | -a | &lt;thread_id&gt; [&gt;replay.txt] <font color="red">dump replay data into a file</font>
   echo [ true | false ] <font color="red">turn on/off command echo mode</font>
-  examine [ address/count ] | [ address,address] <font color="red">show contents of memory from given address</font>
+  examine { address[/count] | address,address } <font color="red">show contents of memory from given address</font>
```

Comment:
It would cover both input variants if to say:
  "show contents of memory range"

Reply:
Ok. Change has been made.